src/pages/give-help/business-support/index.hbs

```diff
@@ -1,29 +1,29 @@
 ---
-title: Partnership - Street Support
+title: Business Support - Street Support
 description: Street support makes it easier to help people facing homelessness. Here's how to get in touch.
 image: https://streetsupport.net/assets/img/og/street-support.png
 layout: default.hbs
 permalink: false
 jsBundle: sponsor
 section: givehelp
-page: partnership
+page: business-support
```

Comment:
leave this as "partnership"

this field is used by metalsmith to inject a class into the body tag. this is then picked up by the css to set the active nav item

Reply:
I see, I've reverted the changes.

Reply:
https://github.com/mikecoram/streetsupport-web/commit/6d67387d6d39f3e29f16ecca341edb971d54d845

https://github.com/mikecoram/streetsupport-web/commit/8d6fd00dfdf0dd8f4d421fce201d2446a537e9a7